lib/utils.js

```diff
@@ -1,92 +1,86 @@
-var _ = require('lodash');
-var corrections = require('../resources/station_corrections');
-var outputFilters = require('../resources/output_filters');
-var props = require('../resources/properties');
-var stations = require('../resources/stations').Stations;
+'use strict';
+const _ = require('lodash');
+const corrections = require('../resources/station_corrections');
+const outputFilters = require('../resources/output_filters');
+const props = require('../resources/properties');
+const stations = require('../resources/stations').Stations;
 
-var changeTypeLookup = {
+const changeTypeLookup = {
   correction: corrections,
   abbreviation: outputFilters['stations']
 };
 
-exports.changeStationName = function (stationName, changeType) {
-  var mapping = changeTypeLookup[changeType];
+exports.changeStationName = (stationName, changeType) => {
+  const mapping = changeTypeLookup[changeType];
   return _.has(mapping, stationName)
     ? mapping[stationName]
     : stationName;
 };
 
-exports.joinListConjunction = function (list, separator, conjuction) {
-  var joinedList = '';
-  if (list.length === 1) {
-    joinedList = list[0];
-  } else {
-    joinedList = list.slice(0, list.length - 1).join(separator) +
-      (list.length > 1
-        ? conjuction + _.last(list)
-        : '');
-  }
-  return joinedList;
+exports.joinListConjunction = (list, separator, conjuction) => {
+  const commaSeparatedWords = list.slice(0, list.length - 1).join(separator);
+  return `${commaSeparatedWords.length
+    ? (`${commaSeparatedWords} ${conjuction} `)
+    : ''}${_.last(list)}`;
```

Comment:
Wouldn't have passed travis if there was a syntax error :wink:.

I can see why last getting appended after the conjunction is unclear. `${_.last(list)}` is not part of the ternary expression.  I'll move it to the next line.